.travis.yml

```diff
@@ -11,9 +11,9 @@ before_install:
   # PRs to master are only ok if coming from dev branch
   - '[ $TRAVIS_PULL_REQUEST = "false" ] || [ $TRAVIS_BRANCH != "master" ] || ([ $TRAVIS_PULL_REQUEST_SLUG = $TRAVIS_REPO_SLUG ] && [ $TRAVIS_PULL_REQUEST_BRANCH = "dev" ])'
   # Pull the docker image first so the test doesn't wait for this
-  - docker pull nfcore/lncpipe
+  - docker pull nfcore/lncpipe:dev
   # Fake the tag locally so that the pipeline runs properly
-  - docker tag nfcore/lncpipe nfcore/lncpipe:latest
+  - docker tag nfcore/lncpipe:dev nfcore/lncpipe:dev
```

Comment:
I'm not sure that you really need to tag the `dev` as `dev`

Reply:
i will update this once released.